tests/database/image/unit/test_image.py

```diff
@@ -41,13 +47,16 @@ def tearDown(self):
 
```

Comment:
> 1st one we need a loop as it is a 2D array, need to think about it more.

Why/How is `image[]` a 2d list? even if that's the case you could do this `image[0][0]`. I still don't understand why we would need for loops.
take your own time to think about. 

> 3rd there is a cascade rule in db

That's nice!